src/ReactTestWrapper.js

```diff
@@ -29,7 +29,8 @@ export default class ReactTestWrapper extends TestWrapper {
   }
 
   attr (name) {
-    return this.el.getAttribute(name) || undefined
+    const value = this.el.getAttribute(name)
+    return value !== null ? value : undefined
```

Comment:
You mean, if the value is empty, we should return `true` instead of the empty string?

`undefined` is only there when the attribute doesn't exist at all.